Source/Core/Core/IOS/USB/Bluetooth/WiimoteDevice.cpp

```diff
@@ -154,7 +154,7 @@ void WiimoteDevice::SetBasebandState(BasebandState new_state)
 
 const WiimoteDevice::SChannel* WiimoteDevice::FindChannelWithPSM(u16 psm) const
 {
-  for (auto& [cid, channel] : m_channels)
+  for ([[maybe_unused]] auto& [cid, channel] : m_channels)
```

Comment:
This is actually considered a GCC bug: https://gcc.gnu.org/bugzilla/show_bug.cgi?id=81767

I think this commit should be dropped as it is fixed in newer GCC versions and it hurts readability.